build.gradle

```diff
@@ -61,30 +72,15 @@ subprojects {
   
   dependencies {
     implementation platform("org.springframework.boot:spring-boot-dependencies:$bootVersion")
+    testImplementation platform("org.junit:junit-bom:${spinnaker.version('jupiter')}")
```

Comment:
This means we can omit the version on all the JUnit dependencies as it gets them from this BOM.